config/webpack-config.js

```diff
@@ -60,27 +60,36 @@ const COMMON_CHUNK_CONFIG = new webpack.optimize.SplitChunksPlugin( {
   name: COMMON_BUNDLE_NAME
 } );
 
+const STATS_CONFIG = {
+  stats: {
+    entrypoints: false
```

Comment:
This is such a short config, what do you think about putting the `false` value in a constant and then putting the rest in each webpack object? This would make the object spread unnecessary and simplify the webpack configs that aren't using multiple copies of `STATS_CONFIG`.

Reply:
I like it as an separate object because it can be extended when debugging and you don't have to update things in multiple places.  This is the entire list of stat options: https://webpack.js.org/configuration/stats/#stats

Reply:
Ah gotcha, I didn't realize there were so many settings in there. One thing I'm confused on is https://webpack.js.org/configuration/stats/#stats shows `entrypoints: false` like that's the default value. What's the default state of the stats object in the config? Is the default that it's absent?

Reply:
yeah, I'm not sure what the default values are and the string value doesn't work with Webpack stream. https://github.com/shama/webpack-stream/issues/163.